src/CRM/CivixBundle/Command/InfoGetCommand.php

```diff
@@ -54,7 +55,13 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     $info->load($ctx);
     $info->get();
 
-    foreach ($info->get()->xpath($input->getOption('xpath')) as $node) {
+    $xpath = $input->getOption('xpath');
+    if (is_null($xpath)) {
+      $help = new HelpCommand();
```

Comment:
1. Is this something you can do by tweaking the `addOption(...)`  data to make the field required?

2. When delegating to subcommands, use something like https://github.com/joomlatools/joomlatools-console/commit/019c5cbfc8c601953562f3a097893a1eeac216ef